src/saga/login.js

```diff
@@ -179,24 +184,23 @@ function* autoConnectCheck (action) {
     } else if (vm && vm.id && vm.status !== 'down') {
       const internalVm = Api.vmToInternal({ vm })
       yield put(updateVms({ vms: [internalVm] }))
-
       yield downloadVmConsole(downloadConsole({ vmId }))
     }
   }
 }
 
-function* initialLoad () {
+function* initialLoad () { // NOTE: This is an in-order fetch, could be made parallel?
   yield put(getAllClusters()) // no shallow
   yield put(getAllHosts())
   yield put(getAllOperatingSystems())
   yield put(getAllTemplates({ shallowFetch: false }))
   yield put(getAllStorageDomains())
   yield put(getAllVnicProfiles())
 
-  yield put(getByPage({ page: 1 })) // first page of VMs list
+  yield put(getByPage({ page: 1 })) // first page of VMs list, sets _loadInProgress_ to false when done
```

Comment:
Please remove this comment, would be confusing with the just merged patch refactoring this.

Reply:
Rebase will address this